src/recipes/browserMentions.js

```diff
@@ -0,0 +1,23 @@
+/* 
+    RECIPE: browserMentions
+    -------------------------------------------------------------
+    Author: Morgan, Malick, Lia, Joel
+    Description: Finding sites that support other browsers but don't include Edge
+*/
+
+void function () {
+    window.CSSUsage.StyleWalker.recipesToRun.push(function browserMentions(element, results) {
+        var str = element.innerText.toLowerCase();
```

Comment:
remember you don't want to run `toLowerCase()` on something that may be `null`. So ensure to check this first before running a method on `str`.